specreduce/wavelength_calibration.py

```diff
@@ -70,6 +70,15 @@ def __init__(self, input_spectrum, matched_line_list=None, line_pixels=None,
         self._potential_wavelengths = None
         self._catalog = catalog
 
+        if not isinstance(input_spectrum, Spectrum1D):
+            raise ValueError('Input spectrum must be Spectrum1D.')
+
+        # make sure spec. is dispersed right to left, should be user's responsibility
+        spec_axis = input_spectrum.spectral_axis.value
+        sorted_increasing = np.all(spec_axis[1:] >= spec_axis[:-1])
+        if not sorted_increasing:
+            raise ValueError('Spectrum must be dispersed left to right.')
```

Comment:
```suggestion
            raise ValueError('Spectral axis values must be increasing from left to right.')
```
I think "dispersed left to right" is potentially ambiguous, may as well say explicitly what we're checking for.